jenkins-pipelines/operator/eks/longevity-scylla-operator-twcs-48h.jenkinsfile

```diff
@@ -0,0 +1,19 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+longevityPipeline(
+    backend: 'k8s-eks',
+    region: 'eu-north-1',
+    test_name: 'longevity_twcs_test.TWCSLongevityTest.test_custom_time',
+    test_config: '''["test-cases/longevity/longevity-twcs-48h.yaml", "configurations/operator/twcs-basic-48h.yaml"]''',
+    email_recipients: 'qa@scylladb.com,scylla-operator@scylladb.com',
+    k8s_enable_performance_tuning: true,
+    availability_zone: 'a,b',
+    post_behavior_db_nodes: 'destroy',
+    post_behavior_loader_nodes: 'destroy',
+    post_behavior_monitor_nodes: 'destroy',
+    post_behavior_k8s_cluster: 'destroy',
+    timeout: [time: 3060, unit: 'MINUTES']
```

Comment:
not needed, timeouts are calculated in getJobTimeouts.groovy

Reply:
Done